TypeError [ERR_INVALID_ARG_TYPE]: The "listener" argument must be of type Function. Received type object

[16:35:49.948] [ERROR]  TypeError [ERR_INVALID_ARG_TYPE]: The "listener" argument must be of type Function. Received type object
    at ClientRequest.once (events.js:288:11)
    at new ClientRequest (_http_client.js:141:10)
    at request (https.js:272:10)
    at Object.get (https.js:276:15)
    at Class.twitterGetFollowers (/home/pi/MagicMirror/modules/MMM-social-counter/node_helper.js:87:8)
    at Class.socketNotificationReceived (/home/pi/MagicMirror/modules/MMM-social-counter/node_helper.js:17:14)
    at Socket.<anonymous> (/home/pi/MagicMirror/js/node_helper.js:113:11)
    at Socket.emit (events.js:182:13)
    at /home/pi/MagicMirror/node_modules/socket.io/lib/socket.js:528:12
    at process._tickCallback (internal/process/next_tick.js:61:11)

--- a/node_helper.js
+++ b/node_helper.js
@@ -85,8 +85,9 @@ module.exports = NodeHelper.create({
   twitterGetFollowers: function (username) {
     https
       .get(
-        `https://api.twitter.com/1.1/users/show.json?screen_name=${username}`,
         {
+          hostname: 'api.twitter.com',
+          path: `/1.1/users/show.json?screen_name=${username}`,
           headers: {
             Authorization: `Bearer ${this.jwt}`,
           },

